homeassistant/components/microbees/climate.py

```diff
@@ -0,0 +1,142 @@
+"""Climate integration microBees."""
+
+from typing import Any
+
+from microBeesPy import Sensor
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+from .entity import MicroBeesActuatorEntity
+
+CLIMATE_PRODUCT_IDS = {76, 78}
```

Comment:
Please add comments to explain the magic number
```suggestion
CLIMATE_PRODUCT_IDS = {
    76,  # Blabla
    78,  # Blabla
}
```

Please also open PRs on the existing microbee platforms which do the same thing, or even better, add constants to the `microBeesPy` library.